src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -1037,6 +1037,74 @@
         az acr token update -n MyToken -r MyRegistry --scope-map MyNewScopeMap
 """
 
+helps['acr agentpool'] = """
+type: group
+short-summary: Manage private Tasks agent pools with Azure Container Registries.
+"""
+
+helps['acr agentpool create'] = """
+type: command
+short-summary: Create an agent pool for an Azure Container Registry.
+examples:
+  - name: Create the agent pool 'MyAgentName' associated with the registry 'MyRegistry'.
+    text: >
+        az acr agentpool create -n MyAgentName -r MyRegistry
+  - name: Create the agent pool 'MyAgentName' with n agent count.
+    text: >
+        az acr agentpool create -n MyAgentName -r MyRegistry --count n
+  - name: Create the agent pool 'MyAgentName' associated with the registry 'MyRegistry' in VNET 'MyVnetId'.
+    text: >
+        az acr agentpool create -n MyAgentName -r MyRegistry --vnet-id MyVnetId
+"""
+
+helps['acr agentpool update'] = """
+type: command
+short-summary: Update an agent pool for an Azure Container Registry.
+examples:
+  - name: Update the agent pool 'MyAgentName' count
+    text: >
+        az acr agentpool update -n MyAgentName -r MyRegistry --tag NewCount
+  - name: Update the agent pool 'MyAgentName' tag
+    text: >
+        az acr agentpool update -n MyAgentName -r MyRegistry --tag MyTag
+"""
+
+helps['acr agentpool delete'] = """
+type: command
+short-summary: Delete an agent pool.
+examples:
+  - name: Delete the agent pool 'MyAgentName'.
+    text: >
+        az acr agentpool delete -n MyAgentName -r MyRegistry
+"""
+
+helps['acr agentpool list'] = """
+type: command
+short-summary: List the agent pools for an Azure Container Registry.
+examples:
+  - name: List agent pools and show the result in a table.
+    text: >
+        az acr agentpool list -r MyRegistry
```

Comment:
Does this show as a table? I believe to show it as a table you need `-o table`

Reply:
good catch thanks